cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/ElytronUtil.java

```diff
@@ -488,6 +488,9 @@ public static ServerSSLContext getServerSSLContext(CommandContext context, Strin
         if (sslContext != null) {
             String kmName = sslContext.get(Util.KEY_MANAGER).asString();
             ModelNode km = getChildResource(kmName, Util.KEY_MANAGER, context);
+            if (km == null) {
+                throw new NullPointerException("The ServerSSLContext " + sslContextName + " references the KeyManager " + kmName + " that doesn't exist.");
```

Comment:
I am wandering if a IllegalStateException would be a better fit than NPE?

Reply:
@jfdenise In earlier fixings regarding NullPointers it was a requirement to have the same Exception as before (IllegalArgumentEx vs NullPointerEx). That was the reason to choose NPE here, too. My point of the fix is to have an "fail early" and a clear message, but I don't have a preference for the type of the exception.

Reply:
I would vote for IllegalArhumentEx, NPE seems strange to me.

Reply:
@jfdenise thanks for your feedback. I'm going to provide an update when the CI is re-opened for everything in Zulip.

Reply:
@jfdenise The changes requested are present now. Could you check and approve, please?